docs/maintainer-guide.md

```diff
@@ -118,6 +135,24 @@ The PM's actions are the same as when [making a release](#making-a-release), min
 
 ## Other tasks
 
+### Security vulnerabilities
+
+Security vulnerabilities, once reported and confirmed, should be treated as a candidate for hot patch (i.e. fixed in the soonest possible time directly on the `release` branch).
+
+Since the detail of such vulnerability cannot be disclosed until it is fixed, an issue can be created just before a PR for the fix is submitted, with minimal information (e.g. simply "Security vulnerability" as an issue with no further description).
+The complete details can be filled in just before merging and/or after the fix is deployed.
```

Comment:
For critical problems, there should not be any info released until the patch is released (because it is possible to guess the vulnerability from the patch). May be ask to email us directly to discuss how to proceed? Most likely critical fixes will have to be done via a private fork.

Reply:
Will reword in the next revision of the issue.